### ISSUE TITLE
Partially-quoted type annotation should generate error

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5811]
The following type annotation generates a runtime error if `from __future__ import annotations` is not used. 

```python
x: "list"[int]
```

Even if evaluation is deferred, this should be considered an illegal form for a type annotation, and pyright should generate an error.

This is related to mypy issue https://github.com/python/mypy/issues/9932.

**Comment by erictraut** [comment]
This is included in pyright 1.1.325, which I just published. It will also be included in an upcoming pylance release.
